test/functionalTest/cases/4113_alarm_for_notification_queue_overpassing_a_given_threshold/alarm_for_notification_queue_overpassing_a_given_threshold.test

```diff
@@ -0,0 +1,292 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+alarm for notification queue overpassing a given threshold
+
+--SHELL-INIT--
+dbInit ${CB_DB_NAME} serv1
+dbInit ${CB_DB_NAME} serv2
+dbInit ${CB_DB_NAME} serv3
+brokerStart CB 0 IPv4 -multiservice -notifAlarmThreshold 90 -notificationMode threadpool:3:2,serv1:3:2,serv2:3:2,serv3:3:2
```

Comment:
I'd suggest to use some slightly different here:

```suggestion
brokerStart CB 0 IPv4 -multiservice -notifAlarmThreshold 50 -notificationMode threadpool:6:1,serv1:4:1,serv2:2:1
```

(No need of serv3, with two of them is enough)

And reformulate steps 05-08 this way:

```
# 0x. Create/update entity in serv1 5 times (update #3 raises alarm)
# 0x. Create/update entity in serv2 3 times (update #2 raises alarm)
# 0x. Create/update entity in serv3 (default) 7 times (update #4 raises alarm)
```

--------

In addition, I'd suggest a variant of this test (eg. alarm_for_notification_queue_overpassing_a_given_threshold_relog.test adding the `-relogAlarm` CLI to the brokerStart, this way:

```suggestion
brokerStart CB 0 IPv4 -multiservice -relogAlarms -notifAlarmThreshold 50 -notificationMode threadpool:6:1,serv1:4:1,serv2:2:1
```

With these steps:

```
# 0x. Create/update entity in serv1 5 times (update #3 raises alarm, update #4 and #5 cause repeated log)
# 0x. Create/update entity in serv2 3 times (update #2 raises alarm, update #3 cause repeated log)
# 0x. Create/update entity in serv3 (default) 7 times (update #4 raises alarm, updates #5, #6 and #7 cause repeated log)
```

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/8e4ea933d2078dce4f7962d4fd9f04ac5785f0be